CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+If you are interested in assuming ownership of a CIP, send a message asking to take over, addressed to both the original author and the CIP editor. If the original author doesn’t respond to email in a timely manner, the CIP editor will make a unilateral decision.
+
+# CIP Editors
+The current CIP editors are:
+`* James Prestwich (@prestwich)`
+`* Yaz Khoury (@YazzyYaz)`
+
+# CIP Editor Responsibilities
+For each new CIP that comes in, an editor does the following:
+* Read the CIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don’t seem likely to get to final status.
+* The title should accurately describe the content.
+* Ensure that the CIP is in line with Celo's [vision and values](https://celo.org/about), and that discussion around it conforms to the [Celo Code of Conduct](https://celo.org/code-of-conduct).
+* Check the CIP for language (spelling, grammar, sentence structure, etc.), markup (GitHub flavored Markdown), code style
+
+If the CIP isn’t ready, the editor will send it back to the author for revision, with specific instructions.
+Once the CIP is ready for the repository, the CIP editor will:
+* Assign a CIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in the Issues section of this repository about this CIP)
+* Merge the corresponding pull request
+* Send a message back to the CIP author with the next step.
+
+Many CIPs are written and maintained by developers with write access to the Celo codebase. The CIP editors monitor CIP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
+
+The editors don’t pass judgment on CIPs. We merely do the administrative & editorial part.
+
+# Style Guide
+When referring to a CIP by number, it should be written in the hyphenated form CIP-X where X is the CIP’s assigned number.
+
+# History
+This document was derived heavily from [EIP-1](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md) written by Hudson Jameson and Martin Becze which in turn was derived from [Bitcoin’s BIP-0001](https://github.com/bitcoin/bips) written by Amir Taaki which in turn was derived from  [Python’s PEP-0001](https://www.python.org/dev/peps/) . In many places text was simply copied and modified. Although the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Celo Improvement Process, and should not be bothered with technical questions specific to Celo or the CIP. Please direct all comments to the CIP editors.
+
+# Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Everything else we do is Apache 2 where possible.

Reply:
Interesting. I normally would go with Apache 2 but I'm using Creative Commons here because improvement proposals aren't code but specifications on code. I've been on calls with Software Freedom Law Center about using CC-BY-SA along with a DCO (in case one is worried about anonymous contributors doing malicious things) for blockchain improvement proposals. However, I don't know what's best and I believe this is something we can ask the legal team. If needed, I think we can even reach out to the Software Freedom Law Center to see what would work best for CIPs.